lib/Bailador/App.pm

```diff
@@ -1,6 +1,6 @@
 use v6.c;
 
-use Log::Any;
+use Log::Any:ver('0.9.4');
```

Comment:
This might be updated in the META6.json as well. I dont know if mi6 does this for us.

Reply:
I hope :)
It could be a pull request in the worst case.